tests/integration/targets/cloudwatchevent_rule/tasks/aliases

```diff
@@ -0,0 +1,2 @@
+cloud/aws
```

Comment:
This file needs to be in the root of the target directory, at `tests/integration/targets/cloudwatchevent_rule/aliases`.
This integration test is not actually running in CI right now because ansible-test doesn't see this file.